codice/alliance
# [0.3.x] CAL-382 Convert NITF digraphs to trigraphs

#### What does this PR do?
Converts NITF digraphs to trigraphs when storing on the catalog taxonomy attributes to be more consistent with the rest of the system.

#### Who is reviewing it? 
@emanns95 
@paouelle 
@vinamartin
@emmberk

#### Choose 2 committers to review/merge the PR.
@clockard
@lessarderic 

#### How should this be tested?
1. Ingest a NITF with digraphs (two letters) for country codes (like owner producer or releasability)
2. Inspect metacard and see that they have been converted to trigraphs (three letters) for Security.OWNER_PRODUCER and Security.RELEASABILITY. Verify corresponding `ext` NITF attributes still have original digraph data.

#### Any background context you want to provide?
Some NITFs were failing to ingest due to security rejecting digraphs in taxonomy security attributes.

#### What are the relevant tickets?
[CAL-382](https://codice.atlassian.net/browse/CAL-382)

#### Screenshots (if appropriate)
#### Checklist:
